src/parcsr_ls/par_interp.c

```diff
@@ -1862,11 +1859,7 @@ hypre_BoomerAMGBuildInterpHE( hypre_ParCSRMatrix   *A,
    hypre_TFree(coarse_counter, HYPRE_MEMORY_HOST);
    hypre_TFree(jj_count, HYPRE_MEMORY_HOST);
    hypre_TFree(jj_count_offd, HYPRE_MEMORY_HOST);
-
-   if (num_procs > 1)
-   {
-      hypre_CSRMatrixDestroy(A_ext);
-   }
+   hypre_CSRMatrixDestroy(A_ext);
```

Comment:
Is this okay because A_ext = NULL when num_procs = 1 (and Destroy checks for NULL)?

Reply:
Exactly... With this, we can save some lines of code...